README.md

```diff
@@ -100,6 +100,16 @@ You can also add an additional service with a different VTR by duplicating the `
 giving it a new name and amending the `PORT` and `VTR` environment variables. The `ports` entry, must reflect the port
 number in the `PORT` environment variable. Each service must have a unique port number.
 
+### Set the analytics environment variables
```

Comment:
Hey @gtvj thanks for your feedback, I've just pushed a commit [here](https://github.com/govuk-one-login/authentication-frontend/pull/1522/commits/b3ac7689f02bc6522d1b9fa8bb6d3a0a08b61a34) but just wanted to test that the env variables set via the instructions on [how to set an env variable](https://github.com/govuk-one-login/authentication-frontend?tab=readme-ov-file#set-the-environment-variables). 

When I run the command `scripts/create-env-file.sh ${target_environment}` using build for example, I can see an error which says `no such file or directory: scripts/create-env-file.sh`, and I think the `create-env-file.sh` does not exist, so i'm not 100% how to test the above feedback, please can you let me know if i've tested this incorrectly?

Reply:
Hi @Joanneyoung01 - I think the file might be missing because the branch is behind `main` and doesn't have the commit that introduced the file. A rebase onto `main` should pick up the commits from the [relevant pull request](https://github.com/govuk-one-login/authentication-frontend/pull/1576). Let me know if I can help with anything.

Reply:
Hey @gtvj thanks for the clarificatio, i've rebased from main and the script is running now - but i'm not able to verify my changes have successfully created the environment variables...  Apologies, please  let me know if i'm missing something. I wondered if I'm carrying out the correct steps - firstly I've amended the .env variables in the [here](https://github.com/govuk-one-login/authentication-frontend/pull/1522/commits/b3ac7689f02bc6522d1b9fa8bb6d3a0a08b61a34), then i've run the script command `scripts/create-env-file.sh ${build}` and I can see this error message...


![Screenshot 2024-04-30 at 18 06 46](https://github.com/govuk-one-login/authentication-frontend/assets/38694448/885c6a7b-a672-4501-93b1-6150bf3ac53e)

Reply:
Hey @gtvj there may be an issue with access rights, as i've raised this [here](https://gds.slack.com/archives/C02KUCT0ZAS/p1714573300903569?thread_ts=1713968574.014229&cid=C02KUCT0ZAS) 

I've made the appropriate changes in the `scripts/_create_env_file.py` file, as explained on the slack thread I'm not able to run the script because what seems to be access rights unfortunately. I was wondering if it would be possible to test on your side to verify these changes? Hoping at some point I'll be able to test this myself in future 🤞

Reply:
Hi, I've tested this and it is working 👍🏽